utilities/src/main/java/io/syndesis/qe/bdd/validation/MqttValidationSteps.java

```diff
@@ -52,9 +53,27 @@ public void sendAndRecieveMessage(String senderTopic, String receiverTopic) {
         }
 
         //check flag
-        Assertions.assertThat(Receiver.RECEIVED_FLAG).as("Message was not received!!!")
+        assertThat(Receiver.RECEIVED_FLAG).as("Message was not received!!!")
```

Comment:
one exclamation mark should be enough :-) (I know you didn't write that message)

Reply:
Actually I did... :D